src/extensions/default/Inline3DParametersEditor/ParameterEditor.js

```diff
@@ -113,7 +112,7 @@ define(function (require, exports, module) {
         this._hideInputs();
 
         this._values = [];
-        for(var i = 0; i<this._numberOfParameters; i++) {
+        for(var i = 0; i < this._numberOfParameters; i++) {
```

Comment:
Unless I'm reading this code wrong, rather than the four lines for declaring an empty array and then doing a for loop, you can do this in a single line:
```
this._values = this.$sliders.map(function(idx, s) { return parseFloat(s.value); });
```
(this can be done in many places, so "I'd rather keep everything for loops" is an acceptable preference)